README.md

```diff
@@ -94,10 +90,10 @@ Launch with `sky launch`:
 ```bash
 sky launch my_task.yaml
 ```
-SkyPilot will perform multiple actions for you:
+SkyPilot performs these heavy-lifting actions for you:
```

Comment:
`SkyPilot performs the heavy-lifting for you:`?

Reply:
Rephrased to `SkyPilot then performs the heavy-lifting for you, including:`, which seems to read better.